tests/lit-tests/1415.ispc

```diff
@@ -0,0 +1,53 @@
+// This test checks that generated code does not try to delete to zero.
```

Comment:
divide by zero?

Reply:
There is still a "delete to zero." in a bunch of lit tests.

Reply:
Indeed, fixed.